### PR TITLE
Replace use of apistar package with local API theme files

### PR DESCRIPTION
**Pull request by amotl**
## Problem
[apistar](https://docs.apistar.com/) is not maintained any longer.

## Details
This patch has already been submitted by @tabotkevin with GH-480, but got lost for whatever reason.

### CODE REVIEW

File: responder/api.py [hunk 1]
```diff
@@ -120,6 +122,7 @@ def __init__(
                 license=license,
                 openapi_route=openapi_route,
                 static_route=static_route,
+                api_theme=api_theme,
```
Comment (tabotkevin) [inline responder/api.py:125]: For clarity, this variable should called `openapi_theme`

File: responder/statics.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
+API_THEMES = ["elements", "rapidoc", "redoc", "swagger_ui"]
 DEFAULT_ENCODING = "utf-8"
-DEFAULT_API_THEME = "swaggerui"
+DEFAULT_API_THEME = "swagger_ui"
```
Comment (tabotkevin) [inline responder/statics.py:3]: and this variable `DEFAULT_OPENAPI_THEME`

**Review by tabotkevin — CHANGES_REQUESTED** [review]
Almost there just minor variable name changes.

File: responder/api.py (hunk 1 above)
Comment (amotl) [reply responder/api.py]: Thanks for the guidance. I used your https://github.com/kennethreitz/responder/commit/0874ce87e98b4f75ba6bcff157c3b437414bd1fa.patch, where it was like `api_theme=api_theme`.

Now, 1107f5d039efdd2f65 adjusts the code according to your suggestions. Is it correct?

File: responder/statics.py (hunk 2 above)
Comment (amotl) [reply responder/statics.py]: 1107f5d039efdd2f65 also renames to `DEFAULT_OPENAPI_THEME`, see comment above.

File: responder/api.py [hunk 3]
```diff
@@ -54,6 +55,7 @@ def __init__(
         cors=False,
         cors_params=DEFAULT_CORS_PARAMS,
         allowed_hosts=None,
+        api_theme=DEFAULT_OPENAPI_THEME,
```
Comment (tabotkevin) [inline responder/api.py:58]: Its still here, everywhere api_theme is used should be replaced with `openapi_theme` and same thing for `DEFAULT_API_THEME` to `DEFAULT_OPENAPI_THEME`

**Review by tabotkevin — CHANGES_REQUESTED** [review]

File: responder/api.py (hunk 3 above)
Comment (amotl) [reply responder/api.py]: Right, another spot has now been fixed, thanks for spotting. 80eebd5b1d0 includes all the renames.

**Review by tabotkevin — APPROVED** [review]
LGTM!

**Pull request merged**